_data/projects/hapi.yml

```diff
@@ -0,0 +1,10 @@
+name: hapi
+desc: A rich framework for building applications and services
+site: http://hapijs.com/
+tags:
+- JavaScript
+- node.js
+- API
+upforgrabs:
+  name: Help Wanted
+  link: http://hapijs.com/contribute
```

Comment:
I'd like for this link to point directly to the list of issues suited for new contributors. The landing page is nice, but it doesn't seem to render for me currently:

<img width="1045" src="https://cloud.githubusercontent.com/assets/359239/12104652/766edec2-b39d-11e5-8aec-e3c1891f30d1.png">

The first "See all" link is really great - and I hadn't thought about using the issue search in that way. We've had some discussions about supporting more flexibility with projects that span multiple repositories (or multiple labels like in #240) - will think further on this...